apps/client/src/shared/component/Header/Notification/index.tsx

```diff
@@ -1,97 +1,80 @@
 import { iconStyle } from "@/shared/component/Header/index.css";
-import { useQueryClient } from "@tanstack/react-query";
-import { AnimatePresence, motion } from "framer-motion";
-import { useRouter } from "next/navigation";
-import { type HTMLAttributes, useState } from "react";
-import NotificationListItem from "./NotificationItem";
+import { useQuery } from "@tanstack/react-query";
+import {} from "framer-motion";
+import { useState } from "react";
 
-interface NotificationProps extends HTMLAttributes<HTMLUListElement> {
-  notificationList: NotificationItem[];
+export enum NotificationType {
+  ALL = "ALL",
+  PROBLEM = "PROBLEM",
+  COMMENT = "COMMENT",
+  STUDY_GROUP = "STUDY_GROUP",
 }
 
-const Notification = ({ notificationList, ...props }: NotificationProps) => {
-  const router = useRouter();
+const notificationMap: Record<NotificationType, string> = {
+  [NotificationType.ALL]: "전체",
+  [NotificationType.PROBLEM]: "문제",
+  [NotificationType.COMMENT]: "코멘트",
+  [NotificationType.STUDY_GROUP]: "스터디",
+};
 
-  const [notifications, setNotifications] = useState(notificationList);
+const Notification = () => {
+  const [notificationType, setNotificationType] = useState<NotificationType>(
+    NotificationType.ALL,
+  );
+  const [isExpanded, setIsExpanded] = useState(false);
 
-  const queryClient = useQueryClient();
-  const { mutate: readNotiMutate } = useReadNotiItemMutation();
-  const { mutate: readAllMutate } = useReadAllNotiMutation();
-  const { mutate: deleteMutate } = useDeleteNotiMutation();
+  const { data } = useQuery(useNotificationsQueryObject(notificationType));
+  const notiCounts = data ? data.filter((item) => !item.isRead).length : 0;
 
-  const handleItemClick = (data: NotificationItem) => {
-    if (!data.isRead) readNotiMutate(data.id);
-    router.push(
-      `/group/${data.groupId}${
-        data.problemId ? `/problem-list/${data.problemId}` : ""
-      }${data.solutionId ? `/solved-detail/${data.solutionId}` : ""}`,
-    );
+  const shrinkList = () => {
+    setIsExpanded(false);
   };
 
-  const handleItemDelete = (notificationId: number) => {
-    deleteMutate(notificationId, {
-      onSuccess: async () => {
-        setNotifications((prev) =>
-          prev.filter((item) => item.id !== notificationId),
-        );
-        await queryClient.invalidateQueries({
-          queryKey: ["notifications"],
-        });
-      },
-    });
+  const expandList = () => {
+    setIsExpanded(true);
   };
 
   return (
     <div className={notificationContainer}>
-      {notificationList.length > 0 ? (
-        <>
-          <button
-            onClick={() => readAllMutate()}
-            className={allReadButtonStyle}
+      <header className={headerStyle} aria-labelledby="notification-title">
+        <h2 className={titleStyle} id="notification-title">
+          알림
+        </h2>
+        <div className={countChipStyle}>{`신규 ${notiCounts}`}</div>
+      </header>
+
+      <ul className={notificationTabListStyle}>
```

Comment:
```suggestion
      <ul role="tablist" className={notificationTabListStyle}>
```
자식인 `<NotificationTab>`이 `role="tab"`을 가지므로 부모인 `<ul>`태그에는 `role="tablist"`이 필요해요.